app/views/layouts/api.html.haml

```diff
@@ -0,0 +1,15 @@
+!!! 5
+= stylesheet_link_tag 'home.css'
```

Comment:
Stylesheet should be within the <head>, right under `= stylesheet_link_tag "application", :media => "all"`